spec/features/guest_user/stores/visitor_visits_different_stores_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe "As a guest, when I visit '/store-name-1'" do
+  it "I see the store name, a list of all items associated to that store, and each items name, price, and 'Add to Cart' button." do
+    stores = create_list(:store, 2)
```

Comment:
Why did you choose to make two stores here? You never use the second store, and always need to call stores.first. This seems like it might be unnecessary complexity.

Reply:
Agree with Anna - its best to only create what's necessary for each test.

Reply:
I see what you did here though, I'll allow it 🎩